images/airflow/2.10.1/bin/airflow-user/post-startup-script-verification

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 echo "Executing post startup script verification"
 
-EXPECTED_AIRFLOW_VERSION="2.10.0"
-EXPECTED_WATCHTOWER_VERSION="3.2.0"
+EXPECTED_AIRFLOW_VERSION="2.10.1"
+EXPECTED_WATCHTOWER_VERSION="3.3.1"
```

Comment:
Bumping this as it is changed in the constraints for this version. We will test logging to make sure it doesn't break. Rafid revamped logging for 2.9 so we expect it to go smoothly